x-pack/filebeat/module/cisco/ios/ingest/pipeline.yml

```diff
@@ -4,6 +4,133 @@ processors:
   - set:
       field: event.ingested
       value: '{{_ingest.timestamp}}'
+  - grok:
```

Comment:
I think its great to remove local JS (edge processing), and we are doing that to most, if not all the packages (fleet integrations). I did this recently for IOS, and I wonder if they look similar. Feel free to see my rewritten pipeline here: https://github.com/elastic/integrations/pull/775/files#diff-3718d0152d23976961a0a82548db5e683231f26be18f04d12bdb2ec799973701

Reply:
Ya it is very similar and I actually had it all broken out into multiple grok processors with the exact same conditions but then I changed it to a single grok since the messages should only match one of the patterns, at least the sample logs that we have.

Reply:
The big thing of why i moved it all to the ingest node was because the issue identified that the current dissect, https://github.com/elastic/integrations/pull/775/files#diff-3718d0152d23976961a0a82548db5e683231f26be18f04d12bdb2ec799973701R25, doesn't handle `cisco.ios.facility` with a `-` in it so it had to be changed to grok instead of dissect which is only available on the ingest node